src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/ShortcutGuideViewModel.cs

```diff
@@ -161,12 +170,17 @@ public int OverlayOpacity
             }
         }
 
+        public string GetSettingsSubPath()
+        {
+            return SettingsConfigFileFolder + "\\" + ModuleName;
```

Comment:
@laviusmotileng-ms does this prevent us from reading existing settings?

Reply:
No, its to allow us to run tests without modifying original settings files. When SettingsConfigFileFolder is left blank, the ViewModel will use the original settings file, this variable should be set to "Test" when the ViewModel is used in a test project so that we can create a temporary test folder to store settings files in.